homeassistant/components/shelly/coordinator.py

```diff
@@ -88,6 +88,7 @@ class ShellyEntryData:
     rest: ShellyRestCoordinator | None = None
     rpc: ShellyRpcCoordinator | None = None
     rpc_poll: ShellyRpcPollingCoordinator | None = None
+    script_events: dict[int, list[str]] | None = None
```

Comment:
```suggestion
    rpc_script_events: dict[int, list[str]] | None = None
```
To be clear that it is only relevant for RPC devices